### ISSUE TITLE
Flux Meter altitude aria-valuetext is missing

### ISSUE THREAD

**Issue by arouinfar** [phetsims/greenhouse-effect#379]
Discovered on main. Related to #374 and #368 

The flux meter altitude is currently being output as a raw, unitless number in the a11y view:
![image](https://github.com/phetsims/greenhouse-effect/assets/8419308/29471806-4440-4e51-aa5c-266c0267ca82)

According to the spec, the aria-valuetext should be of the form:
* Photons screen: {{qualtiative value}} {{above/below cloud}}, e.g. high altitude above cloud
* Layer Model screen: {{qualtitave value}} {{position relative to layers}}, e.g. near surface below layer 1

This appears to be a regression, as the behavior was previously correct.

**Comment by arouinfar** [comment]
Actually, this seems to be a bigger problem. The Greenhouse Gas Concentration slider also lacks the proper aria-valuetext. It should provide the concentration on a qualitative scale, not the raw value.
![image](https://github.com/phetsims/greenhouse-effect/assets/8419308/790741b8-23bf-44ae-9fec-ea234f6550f8)

**Comment by jessegreenberg** [comment]
This regression is the same as #367, a common code change temporarily broke all options for AccessibleValueHandler/AccessibleSlider. It is fixed in main @arouinfar and thank you for finding this!

**Comment by arouinfar** [comment]
Thanks @jessegreenberg! Things look good on main, closing.
